nilearn/input_data/multi_nifti_masker.py

```diff
@@ -36,12 +36,21 @@ class MultiNiftiMasker(NiftiMasker, CacheMixin):
         fine tune the mask extraction.
 
     smoothing_fwhm: float, optional
-        If smoothing_fwhm is not None, it gives the size in millimeters of the
-        spatial smoothing to apply to the signal.
+        If smoothing_fwhm is not None, it gives the size in millimeters of
+        the spatial smoothing to apply to the signal.
 
     standardize: boolean, optional
-        If standardize is True, the time-series are centered and normed:
-        their mean is put to 0 and their variance to 1 in the time dimension.
+        If standardize is True, the time-series are normalized using
+        signal.clean().
```

Comment:
I think we should add more details in this documentation. I would add refer to parameter standardize_strategy for cleaning strategies/options. Something similar.